src/yadism/coefficient_functions/light/g4_nc.py

```diff
@@ -0,0 +1,51 @@
+from ..partonic_channel import RSL
+from . import nlo
+from . import partonic_channel as pc
+
+
+class NonSinglet(pc.LightBase):
+    @staticmethod
+    def LO():
+        """
+        |ref| APFEL
+        """
+
+        # leading order is just a delta function
+        return RSL.from_delta(1.0)
+
+    @staticmethod
+    def NLO():
+        """
+        |ref| hep-ph/9412255
```

Comment:
You could already point to the reference in the bibliography.

https://github.com/NNPDF/yadism/blob/030acf43e6fe2001269b3761c92e639b1fc1a9c7/src/yadism/coefficient_functions/light/g1_nc.py#L21